Jenkinsfile

```diff
@@ -29,7 +27,20 @@ dockerTemplate{
                     }
                 }
 
-                snapshot = true
+                stage('notify'){
+                    def changeAuthor = env.CHANGE_AUTHOR
+                    if (!changeAuthor){
+                        error "no commit author found so cannot comment on PR"
+                    }
+                    def pr = env.CHANGE_ID
+                    if (!pr){
+                        error "no pull request number found so cannot comment on PR"
+                    }
+                    def message = "@${changeAuthor} snapshot ${imageName} is available for testing"
+                    container('clients'){
+                        flow.addCommentToPullRequest(message, pr, 'fabric8io/fabric8-online-docs')
```

Comment:
@rkratky Badger cluster has been retired, and that we were using for deploying things, now we need to move the job to cico and can use dev cluster.

The PR was failing because of error ` No route to host` means not able to connect any openshift cluster as badger cluster which was earlier provisioned has been shut down.